contrib/coredns/policy/setup.go

```diff
@@ -4,10 +4,12 @@ import (
 	"github.com/coredns/coredns/core/dnsserver"
 	"github.com/coredns/coredns/plugin"
 	"github.com/coredns/coredns/plugin/dnstap"
-
+	clog "github.com/coredns/coredns/plugin/pkg/log"
 	"github.com/mholt/caddy"
 )
 
+var log = clog.NewWithPlugin("policy")
```

Comment:
Let's change the way plugin works with logs as separate PR.

Reply:
Yup, cannot agree more.